aws-api/src/main/java/com/amplifyframework/api/aws/AWSApiPlugin.java

```diff
@@ -44,8 +45,12 @@
 import com.amplifyframework.core.model.ModelOperation;
 import com.amplifyframework.util.UserAgent;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
Oops, we want the `androidx` one here, I believe.

OkHttp uses the JetBrains annotations on its APIs, and so this probably got added when you implemented an OkHttp interface, right?

For better or worse, I have been _swapping_ such annotations, as I encounter them, into the AndroidX ones.

OkHttp is itself a vanilla Java library, so they _can't_ depend on the AndroidX annotations.

I feel emotionally that these annotations ought to have better support for Android Studio / Android Lint / Kotlin interop. However, I don't know it any scientifically backed objective way. 😆

Reply:
Fixed.